docs/developer-docs/smart-contracts/best-practices/troubleshooting.mdx

```diff
@@ -14,6 +14,20 @@ When a dapp or canister is running with a high amount of latency, there are some
 
 This document will detail some troubleshooting methods and performance optimizations that developers can use when troubleshooting latency.
 
+## Troubleshooting subnet latency
+
+On subnets with low loads, query calls are answered in about 100 milliseconds and update calls are answered in about 2 seconds.
+
+On subnets with higher loads, your application may experience elevated latencies or reduced accessibility. To get more information about the subnet your canister is running on, view the canister's information on the [ICP dashboard](https://dashboard.internetcomputer.org/canisters).
```

Comment:
We can then maybe add what to look for, which is probably mainly high number for "Million Instructions Executed Per Second" compared to other subnets.

Reply:
Added suggestion